index.html

```diff
@@ -29,8 +29,18 @@
             <li><a class="menu-link" href="#contact-form">Contact</a></li>
           </ul>
         </div>
-      </nav>
-    </header>
+      </nav> 
+    </header> 
+    <div class="mobile-menu-list">
+      <div class="closeImg">
+      <img class="pop-up-close-btn" src="./images/cross.png"  alt="cross">
+      </div>
+      <ul>
```

Comment:
- [ ] please follow the figma design, padding, font size 
- [ ] follow the figma close icon position as well

yours
![Screenshot (536)_LI](https://user-images.githubusercontent.com/65298048/196636656-00f553e9-4d29-4e1d-8f3e-1853a3c41114.jpg)

figma
![Screenshot (535)](https://user-images.githubusercontent.com/65298048/196636720-18264ddb-8b39-42e0-8219-5668a12ad4a8.png)